src/lib/components/Header/Header.tsx

```diff
@@ -1,70 +1,111 @@
+  userActions?: Array<UserAction>;
+  /** If true the user dropdown in headerAccount component is visible. It's visible only if enableLogin is true. */
+  enableDropdown?: boolean;
+  /** If true it concatenates selfcareProduct with productsList */
+  addSelfcareProduct?: boolean;
+};
+
+const selfcareProduct: ProductEntity = {
+  id: 'prod-selfcare',
+  title: 'Area Riservata',
+  productUrl: CONFIG.HEADER.LINK.PRODUCTURL,
+  linkType: 'internal',
+};
+const rootLink: RootLinkType = {
+  label: 'PagoPA S.p.A.',
+  href: CONFIG.HEADER.LINK.ROOTLINK,
+  ariaLabel: 'Link: vai al sito di PagoPA S.p.A.',
+  title: 'Sito di PagoPA S.p.A.',
 };
 
 /** SelfCare Header component */
 const Header = ({
   withSecondHeader,
+  productsList,
+  selectedPartyId,
+  selectedProductId = selfcareProduct.id,
+  partyList,
+  loggedUser,
+  assistanceEmail,
+  enableLogin,
+  userActions,
+  enableDropdown,
+  addSelfcareProduct = true,
   onExitAction = () => window.location.assign(CONFIG.URL_FE.LOGOUT),
-  subHeaderChild,
-}: HeaderProps) => {
-  const { t } = useTranslation();
-  return (
-    <Fragment>
-      <AppBar
-        position="relative"
-        sx={{
-          alignItems: 'center',
-          height: '48px',
-          backgroundColor: 'white',
-          boxShadow: 'none',
-        }}
-      >
-        <Toolbar sx={{ width: { xs: '100%', lg: '90%', minHeight: '48px !important' } }}>
-          {/* <PagoPaMiniIcon viewBox="0 0 80 22" sx={{ width: '80px' }} /> */}
-          <Typography sx={{ fontSize: '14px !important', fontWeight: '700' }}>
-            PagoPA S.p.A
-          </Typography>
-          {onExitAction !== null ? (
-            <Box sx={{ flexGrow: 1, textAlign: 'right' }}>
-              <Button
-                variant="outlined"
-                sx={{
-                  width: '88px',
-                  color: '#0073E6',
-                  height: '32px',
-                  border: 'none',
-                  boxShadow: 'none',
-                  '&:hover': { backgroundColor: 'transparent', border: 'none' },
-                }}
-                onClick={onExitAction}
-              >
-                <Typography sx={{ fontSize: '14px', fontWeight: '700', color: '#0073E6' }}>
-                  {t('common.header.exitButton')}
-                </Typography>
-              </Button>
-            </Box>
-          ) : (
-            ''
-          )}
-        </Toolbar>
-      </AppBar>
-      {withSecondHeader === true ? <SubHeader>{subHeaderChild}</SubHeader> : ''}
-    </Fragment>
-  );
-};
+  onLogin,
+  onSelectedProduct,
+  onSelectedParty,
+  onAssistanceClick = (exitAction) => exitAction(),
+}: HeaderProps) => (
+  <Fragment>
+    <HeaderAccount
+      rootLink={rootLink}
+      loggedUser={loggedUser}
+      onAssistanceClick={() =>
+        onAssistanceClick(() => window.location.assign(buildAssistanceURI(assistanceEmail)))
+      }
+      onLogin={onLogin}
+      onLogout={onExitAction}
```

Comment:
onExitAction is not the onLogout!, the onExitAction is the action that check if we are leaving the application

Here we have to redirect the user to the CONFIG.URL_FE.LOGOUT page

(put the redirect inside the onExitAction)